user/conditions-v1.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Conditions
+layout: en
+---
+
+> This page documents the behavior of conditions `v1`, which is currently
+> available using `conditions: v1` in your `.travis.yml` file. This new
+> version replaces `v0` which is currently the default version, and in the
+> process of being deprecated in favor of `v1`. The documentation for `v0` can
+> be found [here](/user/conditions-v0).
+
+Conditions can be used to filter out, and reject builds, stages, and jobs by
+specifying conditions in your build configuration (your `.travis.yml` file).
+See [Conditional Builds, Stages, and Jobs](/user/conditional-builds-stages-jobs)
+for details.
+
+## Examples
+
+This condition requires the branch name to be master (note for PRs this is the
+base branch name):
+
+```
+branch = master
+```
+
+This requires the tag name to match a regular expression (enclose in slashes
+for more complicated expressions):
+
+```
+tag =~ ^v1
+tag =~ /^(v1|v2)/
+```
+
+This requires the event type to be either `push` or `pull_request`:
+
+```
+type IN (push, pull_request)
+```
+
+This requires the branch name to not be one of several names:
+
+```
+branch NOT IN (master, dev)
+```
+
+This require the sender login name to match a given name (use quotes for
```

Comment:
Sender = commit author?

Reply:
this is true iiuc, though I think it should be defined here

Reply:
Nope, the sender is the person that sends the build request. This can be: 

* the person that does `git push`, 
* the person that opens a pull request, 
* the person that sends an API build request
* the person that creates a cron job (i believe, not 100% sure)

I'm not sure where that should be best documented. It's not currently on API v3 but i believe it might be added soon.

Reply:
Maybe i can add a footnote? do we have such things in the docs?

Reply:
for crons `sender` can actually be a random person with permissions on the repo, or the repo owner (which in turn can be an `org` ... so this actually looks a little buggy)

Reply:
Ah, right - yeah, I was thinking of non-cron, non-automated builds

what if we kept the text the same and instead of `sender` used `build creator`?